app/views/assessment/index.js

```diff
@@ -87,8 +87,7 @@ const Assessment = ({ navigation }) => {
   const AssessmentCancel = () => (
     <TouchableOpacity
       onPress={() => {
-        navigation.goBack();
-        navigation.popToTop();
+        navigation.navigate('Start');
```

Comment:
I realize this was already `Start`, but I'd recommend making route names more specific so there's no risk of duplicates in the tree.
e.g.
`AssessmentStart`

Reply:
perhaps `SelfAssessmentStart`